packages/dynamite/dynamite/lib/src/helpers/dart_helpers.dart

```diff
@@ -159,6 +159,7 @@ const _reservedNames = [
   'super',
   'switch',
   'sync',
+  'Type',
```

Comment:
Can you maybe add it to the e2e escaping test?

Reply:
I thought about this as well, but then we should test all the keywords which I find pointless.